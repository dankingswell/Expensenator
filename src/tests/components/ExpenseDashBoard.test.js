import React from "react"
import {shallow} from "enzyme"
import expenseDashBoardPage from "../../components/ExpenseDashBoardPage"

test("should compile correctly", ()=>{
    const wrapper = shallow(<expenseDashBoardPage />);
    expect(wrapper).toMatchSnapshot();
})