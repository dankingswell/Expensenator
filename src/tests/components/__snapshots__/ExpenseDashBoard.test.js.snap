// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should compile correctly 1`] = `<expenseDashBoardPage />`;
